.github/workflows/linux.yml

```diff
@@ -0,0 +1,84 @@
+---
+name: Linux CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  tests:
+    name: "${{ matrix.env.FLAVOR }} ${{ matrix.env.BOOST }} ${{ matrix.env.COMPILER }}"
+    runs-on: "ubuntu-latest"
+
+    container:
+      image: bondciimages.azurecr.io/ubuntu-1604:build-32869346
+
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - { FLAVOR: cs }
+          - { FLAVOR: cpp-core, BOOST: 1.66.0, COMPILER: clang }
+          - { FLAVOR: cpp-grpc, BOOST: 1.66.0, COMPILER: clang }
+          - { FLAVOR: hs }
+          - { FLAVOR: java }
+
+          - { FLAVOR: cpp-core, BOOST: 1.65.1, COMPILER: clang, CRON_ONLY: true }
```

Comment:
I think the way to do the cron is to have two top-level workflows:

1. A workflow with the current `on` that has combinations without `CRON_ONLY: true`.
2. A workflow with an `on.schedule` that has the combinations with `CRON_ONLY: true`.

`CRON_ONLY` could go away from these workflows. The ZSH scripts never refer to it. It is just used by the Travis CI build script to talk to itself.

Reply:
done